### PR TITLE
quake3e: fix compiling for aarch64

### PR DESCRIPTION
**Pull request by RossComputerGuy**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/games/quake3/quake3e/default.nix [hunk 1]
```diff
@@ -15,7 +15,11 @@
 , copyDesktopItems
 , makeDesktopItem
 }:
-
+let
+  arch =
+    /**/ if stdenv.hostPlatform.isx86_64 then "x64"
```
Comment (pmiddend) [inline pkgs/games/quake3/quake3e/default.nix:20]: Why the `/**/`?

**Comment by pmiddend** [comment]
@ofborg build quake3e

**Comment by pmiddend** [comment]
@RossComputerGuy Have you actually tested running this, or does it just compile?

File: pkgs/games/quake3/quake3e/default.nix (hunk 1 above)
Comment (RossComputerGuy) [reply pkgs/games/quake3/quake3e/default.nix:20]: Just a formatting thing that's used a lot in nixpkgs with if statements.

**Comment by RossComputerGuy** [comment]
> Have you actually tested running this

I got it to run but it wants a CD and I don't have one. Haven't gotten any farther.

File: pkgs/games/quake3/quake3e/default.nix (hunk 1 above)
Comment (pmiddend) [reply pkgs/games/quake3/quake3e/default.nix:20]: Ah, seems so. Okay then

**Review by pmiddend — APPROVED** [review]
Can be merged.

File: pkgs/games/quake3/quake3e/default.nix [hunk 2]
```diff
@@ -52,8 +56,8 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
     make install DESTDIR=$out/lib
-    makeWrapper $out/lib/quake3e.x64 $out/bin/quake3e
-    makeWrapper $out/lib/quake3e.ded.x64 $out/bin/quake3e.ded
+    makeWrapper $out/lib/quake3e.${arch} $out/bin/quake3e
```
Comment (drupol) [inline pkgs/games/quake3/quake3e/default.nix:59]: Just FYI, here's how I did with enemy territory: https://github.com/ExpidusOS/nixpkgs/blob/ff6ba36e9a349ea2bf86470eb3c10f0c9025a22b/pkgs/by-name/et/etlegacy/package.nix#L25

**Pull request merged**